token-lending/js/src/instructions/initLendingMarket.ts

```diff
@@ -1,21 +1,21 @@
 import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
 import { PublicKey, SYSVAR_RENT_PUBKEY, TransactionInstruction } from '@solana/web3.js';
-import { blob, struct, u8 } from 'buffer-layout';
+import { blob, struct, u8 } from '@solana/buffer-layout';
 import { LENDING_PROGRAM_ID, ORACLE_PROGRAM_ID } from '../constants';
-import { publicKey } from '../util';
+import { publicKey } from '@solana/buffer-layout-utils';
 import { LendingInstruction } from './instruction';
 
 interface Data {
     instruction: number;
     owner: PublicKey;
-    quoteCurrency: Buffer;
+    quoteCurrency: Uint8Array;
```

Comment:
1. Uint8Array is more portable (Buffer is a node specific implementation of Uint8Array)
2. Caused compilation errors with TypeScript so I had to remove

Reply:
Yeah, I was originally thinking it made sense to just use Buffer since we already have it from buffer-layout, but that lib bundles it, and we don't need to create buffers here anyway.